app/views/player.py

```diff
@@ -18,10 +18,15 @@ def render_player_info(request, report_tag_form=ReportTagForm(),
 
     game = most_recent_game()
     try:
-        player = request.user.player_set.get(game=game, active=True)
+        player = request.user.get_participant_as_subclass(game)
     except ObjectDoesNotExist:
         return redirect('dashboard')
-    team_score = sum([p.score() for p in Player.objects.filter(role=player.role).all()])
+
+    if player.type == 'Player':
```

Comment:
I wonder if you can do `team_score = sum(Player.objects.filter(role=player.role).values_list('score', flat=True))` if `score` was a property. Or if it just works even as a method.

I don't like else statements like this. Just looks gross. Can we just put `team_score = 0` above the if statement and remove the else?

Reply:
tbh we should check if `team_score` is even a desired statistic players want. maybe we could have something more useful there like "shop points" or even "total stuns/tags". removing it for now until we have more info.